connectors/src/connectors/confluence/lib/confluence_client.ts

```diff
@@ -238,6 +244,11 @@ export class ConfluenceClient {
     })();
 
     if (!response.ok) {
+      statsDClient.increment("external.api.calls", 1, [
```

Comment:
Do we want to also count the OAuth error below as one failed attempt ? That might not be the response that we get when reaching API limits

Reply:
Yes, since we can't assume that Confluence does not account for this call in the rate limit computation.